src/webgpu/shader/execution/expression/binary/af_matrix_vector_multiplication.cache.ts

```diff
@@ -15,7 +15,8 @@ const mat_vec_cases = ([2, 3, 4] as const)
             sparseMatrixF64Range(cols, rows),
             sparseVectorF64Range(cols),
             'finite',
-            FP.abstract.multiplicationMatrixVectorInterval
+            // Matrix-vector multiplication has an inherited accuracy, so abstract is only expected to be as accurate as f32
```

Comment:
Thanks for these comments. They very much clarify the issue for a future reader.